sktime/forecasting/nnetsaucedeepmts.py

```diff
@@ -0,0 +1,390 @@
+        'scp-vine-bb8', 'scp-vine-clayton', 'scp-vine-frank', 'scp-vine-gaussian', 
+        'scp-vine-gumbel', 'scp-vine-indep', 'scp-vine-joe', 'scp-vine-student'
+        - 'scp2-vine-tll', 'scp2-vine-bb1', 'scp2-vine-bb6', 'scp2-vine-bb7', 
+        'scp2-vine-bb8', 'scp2-vine-clayton', 'scp2-vine-frank', 'scp2-vine-gaussian',
+          'scp2-vine-gumbel', 'scp2-vine-indep', 'scp2-vine-joe', 'scp2-vine-student'
+
+    block_size: int.
+        size of block for 'type_pi' in ("block-bootstrap", "scp-block-bootstrap", 
+        "scp2-block-bootstrap").
+        Default is round(3.15*(n_residuals^1/3))
+
+    replications: int.
+        number of replications (if needed, for predictive simulation). Default
+          is 'None'.
+
+    kernel: str.
+        the kernel to use for residuals density estimation (used for predictive 
+        simulation). Currently, either 'gaussian' or 'tophat'.
+
+    agg: str.
+        either "mean" or "median" for simulation of bootstrap aggregating
+
+    seed: int.
+        reproducibility seed for nodes_sim=='uniform' or predictive simulation.
+
+    backend: str.
+        "cpu" or "gpu" or "tpu".
+
+    verbose: int.
+        0: not printing; 1: printing
+
+    show_progress: bool.
+        True: progress bar when fitting each series; False: no progress bar when 
+        fitting each series
+
+    """
+    _tags = {
+        # scitype:y controls whether internal y can be univariate/multivariate
+        # if multivariate is not valid, applies vectorization over variables
+        "scitype:y": "both",
+        # fill in "univariate" or "both"
+        #   "univariate": inner _fit, _predict, receives only single-column DataFrame
+        #   "both": inner _predict gets pd.DataFrame series with any number of columns
+        #
+        # specify one or multiple authors and maintainers, only for sktime contribution
+        "authors": ["thierrymoudiki"],  # authors, GitHub handles
+        "maintainers": ["thierrymoudiki"],  # maintainers, GitHub handles
+        # author = significant contribution to code at some point
+        #     if interfacing a 3rd party estimator, ensure to give credit to the
+        #     authors of the interfaced estimator
+        # maintainer = algorithm maintainer role, "owner" of the sktime class
+        #     for 3rd party interfaces, the scope is the sktime class only
+        # remove maintainer tag if maintained by sktime core team
+        #
+        # do not change these:
+        # (look at advanced templates if you think these should change)
+        "y_inner_mtype": "pd.DataFrame",
+        "X_inner_mtype": "pd.DataFrame",
+        "ignores-exogeneous-X": False,
+        "capability:pred_int": True,
+        "capability:pred_int:insample": True,
+        "requires-fh-in-fit": False,
+    }
+
+    # todo: add any hyper-parameters and components to constructor
+    def __init__(
+        self,
+        obj,
```

Comment:
It's not a requirement, but usually we encourage to have no mandatory positional arguments so that users can simply initiate with `NnetsauceDeepMTS()`. Since you need an input here, may I request you to give a more descriptive name of the argument (e.g. regressor), and add few examples in the docstring?